docs/rules/strict.md

```diff
@@ -1,12 +1,12 @@
-# Strict Mode (strict)
+# Strict-Mode Directives (strict)
```

Comment:
Let's keep using "strict mode" (no dash) throughout, as I don't usually see it hyphenated.